en/docs/learn/provisioning-users-to-salesforce.md

```diff
@@ -0,0 +1,150 @@
+                        </markForSuspension>
+                    </http>
+                </endpoint>
+            </call>
+        </sequence>
+    ```
+    Replace `username`, `password`, `clientId`, `clientSecret` as per your created application.
+
+5.  Create the below API from `API > Add API`
+    ```xml
+        <api xmlns="http://ws.apache.org/ns/synapse" name="salesforcerest" context="/salesforce">
+           <resource methods="POST">
+              <inSequence>
+                 <property name="uri.var.email" expression="json-eval($.emails[0])" scope="default" type="STRING"/>
+                 <sequence key="retrieve"/>
+                 <switch source="json-eval($.totalSize)">
+                    <case regex="0">
+                       <payloadFactory media-type="json">
+                          <format>{"sObject":"Lead", "fieldAndValue": {"LastName" : "$1", "Company" : "Guardio", "Email" : "$1", "LeadSource" : "Guardio brochure download."}}</format>
+                          <args>
+                             <arg evaluator="xml" expression="get-property('uri.var.email')"/>
+                          </args>
+                       </payloadFactory>
+                       <sequence key="create"/>
+                    </case>
+                 </switch>
+                 <respond/>
+              </inSequence>
+              <outSequence/>
+              <faultSequence/>
+           </resource>
+        </api>
+    ```
+
+
+##  Configuring the Identity Server
+
+1.  Download the [WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
+2.  Add an Identity Provider to configure outbound provisioning as shown in the below diagram.
+    ![configure-salesforce-idp](../assets/img/tutorials/configure-salesforce-idp.png)
+3.  In the `SCIM Provisioning Configuration`, provide the REST endpoint details of the EI server. If they are secured
+ APIs, provide the credentials as well.
+4.  Configure the `Resident Service Provider` to register the newly created Identity Provider for Outbound
+Provisioning as shown below.
+    ![configure-resident-sp-for-salesforce](../assets/img/tutorials/configure-resident-sp-for-salesforce.png)
+
+
+##  Try Out
+
+1.  Sends a simple SCIM request to the Identity Server similar to following.
+    ```
+        curl -v -k --user admin:admin --data '{"schemas":[],"userName":"nipunt","password":"admin123",
+        "emails":["nipunt@wso2.com"]}' --header "Content-Type:application/json" https://localhost:9443/scim2/Users
+    ```
+    In above, the **email** attribute will be the one that is consumed by the EI Salesforce Connector to create the
+    `Lead` in the Salesforce. You can also send other parameters in this SCIM request and change the EI REST API
+    appropriately to capture the information and send to Salesforce.
+2.  You can see the local user by searching the string `nipun` in the `Management Console > Users and Roles > Users >
```

Comment:
Please change the name here also. 

**Management Console > Users and Roles > Users**